<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1096</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="base_datos_ventana">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1131</width>
      <height>631</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Inicio</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Inventario</string>
     </attribute>
     <widget class="QTableWidget" name="table_product">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>801</width>
        <height>391</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="rowCount">
       <number>10</number>
      </property>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column>
       <property name="text">
        <string>CODIGO</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>DESCRIPCION</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>MEDIA</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>EXISTENCIA</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PRECIO(costo)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PRECIO(publico)</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>416</width>
        <height>57</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ventana Principal</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_list">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>500</y>
        <width>93</width>
        <height>33</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Listar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_update">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>500</y>
        <width>111</width>
        <height>33</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Actualizar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_create">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>500</y>
        <width>93</width>
        <height>33</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Crear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_read">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>500</y>
        <width>126</width>
        <height>33</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Seleccionar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_delete">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>500</y>
        <width>93</width>
        <height>33</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>-10</y>
        <width>1111</width>
        <height>821</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.512, y1:0, x2:0.532, y2:1, stop:0.174129 rgba(232, 173, 35), stop:1 rgba(255, 255, 255, 255));</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label_4</zorder>
     <zorder>table_product</zorder>
     <zorder>label</zorder>
     <zorder>btn_list</zorder>
     <zorder>btn_update</zorder>
     <zorder>btn_create</zorder>
     <zorder>btn_read</zorder>
     <zorder>btn_delete</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Ventas</string>
     </attribute>
     <widget class="QTabWidget" name="base_de_compras">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>841</width>
        <height>521</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="Ventas">
       <attribute name="title">
        <string>Ventas</string>
       </attribute>
       <widget class="QPushButton" name="btn_list_4">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>420</y>
          <width>93</width>
          <height>33</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Listar</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>90</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Modern No. 20</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Nombre</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>130</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>210</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>170</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>NIT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Consumidor Final</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>341</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>MS Reference Sans Serif</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Ventas de Productos</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Modern No. 20</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Codigo</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>10</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>200</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Modern No. 20</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Fecha</string>
        </property>
       </widget>
       <widget class="QCalendarWidget" name="calendarWidget">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>230</y>
          <width>311</width>
          <height>191</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>250</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Modern No. 20</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Direccion</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_2">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>320</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Mayoreo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Minorista</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QWidget" name="Compras">
       <attribute name="title">
        <string>Compras</string>
       </attribute>
       <widget class="QPushButton" name="btn_guardar_compra">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>400</y>
          <width>93</width>
          <height>33</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Guardar</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Modern No. 20</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Codigo</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="text_codigo_compra">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>100</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>381</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>MS Reference Sans Serif</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Compras del Productos</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="text_precio_costo_compra">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>290</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="text_precio_publico_compra">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>350</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="text_descripcion_compra">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>140</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="text_existencia_compra">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>240</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="text_media_compra">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>190</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>200</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Modern No. 20</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Media</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Modern No. 20</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Descripcion</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>290</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Modern No. 20</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Precio</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>240</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Modern No. 20</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Existencia</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>350</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Modern No. 20</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Precio</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>290</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Modern No. 20</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>(Costo)</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>350</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Modern No. 20</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>(Publico)</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tabla">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>160</y>
          <width>461</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Recursos Humanos</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>Financiero </string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
